packages/block-editor/src/components/global-styles/index.js

```diff
@@ -0,0 +1,7 @@
+export {
+	useGlobalStylesReset,
+	useGlobalSetting,
+	useGlobalStyle,
+} from './hooks';
+export { useGlobalStylesOutput } from './use-global-styles-output';
+export { GlobalStylesContext } from './context';
```

Comment:
Do we want to give the whole context object as an API or just the "provider" for external consumers.

Reply:
There's a few places where we need to read the raw context value e.g. variations.

https://github.com/WordPress/gutenberg/blob/64c69123ea57a3c342475c8a72d274d4cf9764a3/packages/edit-site/src/components/global-styles/screen-style-variations.js#L45